src/libs/SearchUIUtils.ts

```diff
@@ -596,6 +593,10 @@ function getAction(data: OnyxTypes.SearchResults['data'], allViolations: OnyxCol
         return CONST.SEARCH.ACTION_TYPES.PAY;
     }
 
+    if (isClosedReport(report)) {
+        return CONST.SEARCH.ACTION_TYPES.DONE;
+    }
+
```

Comment:
@grgia  It's working now, because `canIOUBePaid` returns true but I'm still seeing the report has `stateNum: 2` and `statusNum: 2`, which means closed report. Is that expected?

Reply:
> I'm still seeing the report has stateNum: 2 and statusNum: 2, which means closed report

Hmm. No. They should be Submitted `1, 1` you're right. I mostly wanted to add automated tests. I'll look into the BE error with report state. Not sure where in the flow it's changing to 2,2

Reply:
https://expensify.slack.com/archives/C03TQ48KC/p1749715570694179

Reply:
@hungvu193 2,2 is correct. All reports with No Approvers are Submit&Close. They show the reimbursement button on the closed report on OldDot

Reply:
@grgia  Thanks for confirming. Approving

Reply:
We will likely start using different logic to determine pay buttons. Linking doc for later reference https://docs.google.com/document/d/1gUDVqDdDj0jVOIuFBqPr0D6-vnClB-iztl38PzsiBqE/edit?tab=t.0

Reply:
Example to illustrate